integration-tests/prod_test_manager.py

```diff
@@ -0,0 +1,303 @@
+#         # replace testng source
+#         replace_file(testng_source, testng_destination)
+#         # replace testng server mgt source
+# replace_file(testng_server_mgt_source, testng_server_mgt_destination)
+
+
+def get_db_meta_data(argument):
+    switcher = DB_META_DATA
+    return switcher.get(argument, False)
+
+
+
+def main():
+    try:
+        global logger
+        global dist_name
+
+        logger = cm.function_logger(logging.DEBUG, logging.DEBUG)
+
+        print(str(sys.version_info))
+        if sys.version_info < (3, 6):
+            raise Exception(
+                "To run run-intg-test.py script you must have Python 3.6 or latest. Current version info: " + sys.version_info)
+        cm.read_property_files()
+        if not cm.validate_property_readings():
+            raise Exception(
+                "Property file doesn't have mandatory key-value pair. Please verify the content of the property file "
+                "and the format")
+
+
+        # get properties assigned to local variables
+        pom_path = DIST_POM_PATH[cm.product_id]
+        engine = cm.db_engine.upper()
+        db_meta_data = get_db_meta_data(engine)
+        distribution_path = DISTRIBUTION_PATH[cm.product_id]
+
+
+        # construct database configuration
+        cm.construct_db_config(db_meta_data)
+        # clone the repository
+        cm.clone_repo()
+        # set the custom testng.xml or the product testng.xml
+        #set_custom_testng()
+
+        if cm.test_mode == "RELEASE":
+            dist_name = cm.get_dist_name(pom_path)
+            cm.get_latest_released_dist()
+        elif cm.test_mode == "SNAPSHOT":
+            dist_name = cm.get_dist_name(pom_path)
+            # build the distribution for snapshot test mode
+            cm.build_snapshot_dist(distribution_path)
+        elif cm.test_mode == "WUM":
+            # todo after identify specific steps that are related to WUM, add them to here
+            dist_name = cm.get_dist_name(pom_path)
+            logger.info("WUM specific steps are empty")
+
+        # populate databases
+        db_names = configure_product()
+        if db_names is None or not db_names:
+            raise Exception("Failed the product configuring")
+
+        cm.setup_databases(db_names, db_meta_data)
+        if cm.product_id == "product-apim":
+            module_path = Path(cm.workspace + "/" + cm.product_id + "/" + 'modules/api-import-export')
+            cm.build_module(module_path)
+        intg_module_path = Path(cm.workspace + "/" + cm.product_id + "/" + INTEGRATION_PATH[cm.product_id])
+        cm.build_module(intg_module_path)
+        save_test_output()
+        cm.create_output_property_fle()
+    except Exception as e:
+        logger.error("Error occurred while running the run-intg.py script", exc_info=True)
+    except BaseException as e:
+        logger.error("Error occurred while doing the configuration", exc_info=True)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
need new line at eof.

Reply:
Fix for new line in 5b5b94186229d940cfe147d6a2a8071b0d0320de